src/components/pollutedLocationCard/PollutedLocationCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { BsTrashFill } from "react-icons/bs";
+import { BsTrash } from "react-icons/bs";
+import { RiLogoutCircleRFill } from "react-icons/ri";
+import PollutedLocationDTO from "../../types/backEnd/PollutedLocationDTO";
+
+type PollutedLocation = PollutedLocationDTO & {
+  title?: string;
+};
+
+const PollutedLocationCard: React.FC<PollutedLocation> = ({
+  title,
+  radius,
+  progress,
+}) => {
+  return (
+    <div className="mx-auto bg-white rounded-xl shadow-lg flex items-center space-x-4 sm:w-1/4 md:w-4/5 lg:w-1/4 px-2 pb-2 pt-2">
+      <div className="flex flex-col w-full">
+        <div className="flex flex-row space-x-4">
+          <div className="w-1/2 font-bold">
+            <h1>{title || "Title is missing"}</h1>
+          </div>
+          <div className="w-1/2">
+            <div className="flex justify-between mb-1">
+              <span className="text-sm font-medium text-blue-700 dark:text-white">
+                {progress || 0}%
+              </span>
+              <div className="w-full bg-gray-200 rounded-full h-2.5 dark:bg-gray-700 mt-1.5">
+                <div
+                  className="bg-blue-600 h-2.5 rounded-full"
+                  style={{ width: progress + "%" }}
```

Comment:
What if `progress` is `undefined` here? Probably you need something similar to line 26: `{progress || 0}`